mjml/templates/test.mjml

```diff
@@ -1,11 +1,31 @@
   <mjml>
-    <mj-body>
+    <mj-body background-color="#fff">
+
+      <mj-section background-color="#ffffff" padding-bottom="20px" padding-top="20px">
+       <mj-column>
+         <mj-image src="../../webui/public/img/logo@2x.png" alt="Haven GRC Company Logo" width="82px" height="71px"></mj-image>
```

Comment:
I'm curious about these src paths, I suspect we will need to use a ftl template code that will get us full keycloak domain name, since the resulting email is going to be rendered on a client computer and won't know what domain haven is hosted on.